src/shared/utils/sortedData.js

```diff
@@ -0,0 +1,22 @@
+const getSortedData = (products, state) => {
+  let updatedProducts = [...products];
+
+  function price(originalPrice, discount) {
+    return parseInt(originalPrice) - parseInt((originalPrice * discount) / 100);
+  }
+  if (state.sortBy === "HIGH_TO_LOW") {
+    updatedProducts.sort(
+      (a, b) =>
+        price(b.original_price, b.discount_percent) -
+        price(a.original_price, a.discount_percent)
+    );
+  } else if (state.sortBy === "LOW_TO_HIGH") {
+    updatedProducts.sort(
+      (a, b) =>
+        price(a.original_price, a.discount_percent) -
+        price(b.original_price, b.discount_percent)
+    );
```

Comment:
Suggestion: You can create a utility function for
(a, b) =>
        price(b.original_price, b.discount_percent) -
        price(a.original_price, a.discount_percent)


(a, b) =>
        price(a.original_price, a.discount_percent) -
        price(b.original_price, b.discount_percent)

Reply:
I don't think so we can make a utility function for whole . Or else it won't be re-usable.